readme.md

```diff
@@ -58,18 +58,18 @@ This guide is about runnning the client from source repository. For a [ready-mad
 Clone this repository locally:
 
     git clone https://github.com/FAForever/client.git faf-client
-    
+
 Create a python virtualenv for installing its dependencies:
 
     virtualenv2 ./faf-client-venv --system-site-packages
     ./faf-client-venv/bin/pip install -r ./faf-client/requirements.txt
 
-Now download the `uid` executable:
+Now download the `faf-uid` executable:
 
-    wget https://github.com/FAForever/uid/releases/download/v2.1.0/uid -O ./faf-client/lib/uid
-    chmod +x ./faf-client/lib/uid
+    wget https://github.com/FAForever/uid/releases/download/v2.1.0/uid -O ./faf-client/lib/faf-uid
```

Comment:
Maintainers:

Is `https://github.com/FAForever/uid/releases/download/v2.1.0/uid` going to be renamed at some point to `faf-uid`?